project/Dependencies.scala

```diff
@@ -25,7 +25,7 @@ object Dependencies {
   val commonsHttpClient = "commons-httpclient" % "commons-httpclient" % "3.1"
   val commonsIo = "commons-io" % "commons-io" % "2.4"
   val cssParser = "net.sourceforge.cssparser" % "cssparser" % "0.9.18"
-  val contentApiClient = "com.gu" %% "content-api-client" % "7.27"
+  val contentApiClient = "com.gu" %% "content-api-client" % "7.28"
```

Comment:
Necessary to include packages related bugfix https://github.com/guardian/content-api-scala-client/blob/master/CHANGELOG.md#728